### DISCUSSION TITLE
Enums in Motoko (tweet)

### DISCUSSION THREAD

**Discussion by ggreif** [dfinity/motoko#4834] [Show and tell]
Enumerated types are a simple form of variants where each of the alternatives carries trivial (unit, `()`) payload. The syntax allows to eliminate the payload specification entirely (and it is recommended to do so):
``` Motoko
type Colour = { #red; #green; #blue };
let favourite : Colour = #green;
```

**Comment by jessiemongeon1** [comment]
Scheduled for Friday June 27 2025
